db.json

```diff
@@ -1530,6 +1530,1462 @@
             "is_remote": false,
             "location": "6-r2",
             "event_type": "talk"
+        },
+        {
+            "id": 11,
```

Comment:
too much mock data. Are they really necessary?

Reply:
Agree, will be updated.